depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version=1.0
+
+$(package)_download_path=https://github.com/Christewart/rapidcheck/releases/download/1.0
+
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+
+$(package)_sha256_hash=c228dc21ec24618bfb6afa31d622d1f4ea71168f04ee499e1ffcfc63cd5833f4
```

Comment:
No worries, I don't understand cpp or the build system either. I can take a look later today... I think the issue is **within** the package you uploaded.